src/hotspot/share/prims/jvmtiThreadState.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2003, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Nit: No need in the copyright update anymore.

Reply:
Fixed.